examples/tutorials/hybrid_demucs_tutorial.py

```diff
@@ -0,0 +1,360 @@
+"""
+Hybrid Demucs Music Separation
+==============================
+
+**Author**: `Sean Kim <https://github.com/skim0514>`__
+
+This tutorial shows how to use the Hybrid Demucs model in order to
+perform music separation
+
+"""
+
+
+######################################################################
+# 1. Overview
+# -----------
+# 
+# Performing music separation is composed of the following steps
+# 
+# 1. Build the Hybrid Demucs pipeline.
+# 2. Format the waveform into chunks of expected sizes and loop through
+#    chunks (with overlap) and feed into pipeline.
+# 3. Collect output chunks and combine according to the way they have been
+#    overlapped.
+# 
+# The `Hybrid Demucs <https://arxiv.org/pdf/2111.03600.pdf>`__ model is a developed version of the
+# `Demucs <https://github.com/facebookresearch/demucs>`__ model, a
+# waveform based model which separates music into its
+# respective sources, such as vocals, bass, and drums. Hybrid Demucs effectively uses spectrogram to learn
+# through the frequency domain and also moves to time convolutions.
+# 
+
+
+######################################################################
+# 2. Preparation
+# --------------
+# 
+# First, we install the necessary dependencies. The first requirement is
+# ``torchaudio`` and ``torch``
+# 
+
+import torch
+import torchaudio
+
+print(torch.__version__)
+print(torchaudio.__version__)
+
+
+######################################################################
+# In addition to ``torchaudio``, ``mir_eval`` is required to perform
+# signal-to-distortion ratio (SDR) calculations. ``mir_eval`` can be
+# downloaded utilizing ``pip3`` as seen in the next cell.
+# 
+
+from IPython.display import Audio
+from torchaudio.utils import download_asset
+import matplotlib.pyplot as plt
+
+# To download mir_eval run !pip3 install mir_eval
+from mir_eval import separation
+
+try:
+    import google.colab
+
+    print(
+        """
+        To enable running this notebook in Google Colab, install nightly
+        torch and torchaudio builds and the requisite third party libraries by
+        adding the following code block to the top of the notebook before running it:
+
+        !pip3 uninstall -y torch torchvision torchaudio
+        !pip3 install --pre torch torchaudio --extra-index-url https://download.pytorch.org/whl/nightly/cpu
+        """
+    )
+except ModuleNotFoundError:
+    pass
```

Comment:
this `try ~ except` block no longer makes sense. This was supposed to print a better message in case users did not install nightly. See https://github.com/pytorch/audio/issues/2336 for the context.

The `try` part should be wrapping what we expect might be missing from Google Colab environment. That is `mir-eval` and new `torchaudio.prototype` module. Then the error message have to cover instruction for installing both.